scripts/karma.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+'use strict';
+
+const { Server, stopper } = require('karma');
+const { parseConfig } = require('karma').config;
+const { join } = require('path');
+
+const configFile = join(__dirname, '..', 'integration.conf.js');
+const integrationTests = join(__dirname, '..', 'test', 'integration', 'spec');
+
+// NOTE(mroberts): We have a memory leak, either in twilio-video.js or in
+// Firefox, that causes Firefox to slow down after running a bunch of tests that
+// exercise WebRTC APIs. To workaround this, we spawn Karma for each integration
+// test module.
+async function main() {
+  for (const file of [
```

Comment:
@markandrus Can this list be generated from the folder structure using `require('fs').readDir`?